src/CHANGELOG.tsx

```diff
@@ -15,11 +15,13 @@ import {
   ToppleTheNun,
   Trevor,
   Vireve,
+  bdfreeman1421,
 } from 'CONTRIBUTORS';
 import { ItemLink } from 'interface';
 import SpellLink from 'interface/SpellLink';
 
 export default [
+  change(date(2023, 2, 23), 'Inital Classic Survival Hunter Abilities.', bdfreeman1421),
```

Comment:
Remove this changelog entry since there's one in the hunter changelog file.

Reply:
Done